lib/phoenix_token_auth/mailer.ex

```diff
@@ -25,6 +25,7 @@ defmodule PhoenixTokenAuth.Mailer do
   Both config fields have to be functions returning binaries. welcome_email_subject receives the user and
   welcome_email_body the user and confirmation token.
   """
+  def send_welcome_email(user = %{email: email}, confirmation_token) when email == nil do end
```

Comment:
No worries... the more comments, the more i fix things :)

Its because of this code on users_controller:
```
if changeset.valid? do
      case Util.repo.transaction fn ->
        user = Util.repo.insert(changeset)
        Mailer.send_welcome_email(user, confirmation_token)
      end do
        {:ok, _} -> json conn, :ok
      end
    else
      Util.send_error(conn, Enum.into(changeset.errors, %{}))
    end
```

So, it calls the mailer.send_welcome_email, but if email is nil, it just does nothing.